src/HDWallet.cpp

```diff
@@ -33,9 +33,12 @@ HDNode getMasterNode(const HDWallet& wallet, TWCurve curve);
 const char* curveName(TWCurve curve);
 } // namespace
 
+const size_t MnemonicBufLength = Mnemonic::MaxWords * 10;
```

Comment:
```suggestion
const size_t MnemonicBufLength = Mnemonic::MaxWords * 10;
```

what is the magic number 10?

Reply:
1627711012

Reply:
done, thx

Reply:
0xa36fD8fF0524f287422337d99bABdf05923f581B